src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt

```diff
@@ -173,6 +173,10 @@ class DefaultEthereumMethods(
 
     private fun getChainSpecificMethods(chain: Chain): List<String> {
         return when (chain) {
+            Chain.ETHEREUM__GOERLI, Chain.ETHEREUM__MAINNET -> listOf(
```

Comment:
I think we need to add all ETHEREUM__* networks , or maybe it relevant to all evm based networks?